pyani_plus/public_cli.py

```diff
@@ -820,13 +821,21 @@ def list_runs(
 
 
 @app.command()
-def export_run(  # noqa: C901
+def export_run(  # noqa: C901, PLR0912, PLR0915
     database: REQ_ARG_TYPE_DATABASE,
     outdir: REQ_ARG_TYPE_OUTDIR,
     run_id: Annotated[
         int | None,
         typer.Option(help="Which run to report (optional if DB contains only one)"),
     ] = None,
+    # Would like to replace this with Literal["md5", "filename", "stem"] once typer updated
+    label: Annotated[
+        str,
+        typer.Option(
+            click_type=click.Choice(["md5", "filename", "stem"]),
+            help="How to label the genomes",
+        ),
+    ] = "md5",
```

Comment:
It doesn't seem appropriate to define an Enum just for the command line choices (but that is reasonable for things like ANIm or sourmash mode). There is an open PR on typer which would add support for using ``Literal["md5", "filename", "stem"]`` which is supported in other type-based CLI libraries.

Reply:
Also, should the default become "stem" rather than "md5"?

If so, the easy change for the tests is to explicitly request MD5 labeling. However, there is some readability value in switching to using stem in the expected matrix output fixtures.